src/Resources/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,113 @@
+        $app1 = Get-AzADApplication -DisplayName $appName1
+        Assert-NotNullOrEmpty $app1
+        Assert-AreEqual $replyUrl1 $app1.Web.RedirectUri
+        Assert-AreEqual $homepage1 $app1.Web.HomePageUrl
+        Assert-AreEqual 'AzureADMultipleOrgs' $app1.SignInAudience
+    
+        Assert-AreEqual $app1.Id (Get-AzADApplication -ObjectId $app1.id -Select Id).Id
+        Assert-AreEqual $app1.Id (Get-AzADApplication -ApplicationId $app1.AppId -Select Id).Id
+    
+        $app1Update = Update-AzADApplication -ObjectId $app1.Id -ReplyUrl $replyUrl2 -HomePage $homepage2 -AvailableToOtherTenants $false
+        $app1Update = Get-AzADApplication -DisplayName $appName1
+
+        Assert-AreEqual $replyUrl2 $app1Update.Web.RedirectUri
+        Assert-AreEqual $homepage2 $app1Update.Web.HomePageUrl
+        Assert-AreEqual 'AzureADMyOrg' $app1Update.SignInAudience
+        Assert-NotNullOrEmpty (Get-AzADAppCredential -ObjectId $app1.Id)
+        $pw = New-AzADAppCredential -ObjectId $app1.Id -StartDate (get-date)
+    
+        $certFile = Join-Path $PSScriptRoot 'msgraphtest2.cer'
+        $content = get-content $certFile -AsByteStream
+        $certvalue = [System.Convert]::ToBase64String($content)
+        $cert = New-AzADAppCredential -ObjectId $app1.Id -CertValue $certvalue
+    
+        Remove-AzADAppCredential -ObjectId $app1.Id -KeyId $pw.KeyId
+    
+        $sp1 = New-AzADServicePrincipal -ApplicationId $app1.AppId
+        $sp1 = Get-AzADServicePrincipal -ApplicationId $app1.AppId
+        $sp2 = New-AzADServicePrincipal -DisplayName $spName2        
+        $sp2 = Get-AzADServicePrincipal -DisplayName $spName2
+        $app2 = Get-AzADApplication -DisplayName $spName2
+    } finally {
+        if ($sp1) {
+            Remove-AzADServicePrincipal -ServicePrincipalName $sp1.ServicePrincipalName[0]
+        }
+        if ($sp2) {
+            Remove-AzADServicePrincipal -ObjectId $sp2.Id
+        }
+        if ($app1) {
+            Remove-AzADApplication -DisplayName $appName1
+        }
+        if ($app2) {
+            Remove-AzADApplication -DisplayName $spName2
+        }
+    }
+}
+
+Invoke-LiveTestScenario -Name "Test Group Member" -Description "Test the process of create groups and members." -NoResourceGroup -PowerShellVersion Latest -ScenarioScript `
+{
+    try {
+        $today = (Get-Date).tostring('yyyy-MM-dd')
+        $groupName1 = $today + 'testgroup' + (New-LiveTestRandomName)
+        $groupName2 = $today + 'testgroup' + (New-LiveTestRandomName)
+        $userName = $today + 'testuser' + (New-LiveTestRandomName)
+        
+        $groupMailNickName1 = New-LiveTestRandomName
+        $groupMailNickName2 = New-LiveTestRandomName
+        $userMailNickName = New-LiveTestRandomName
+        $userPrincipalName = $userMailNickName + 'microsoft.com#EXT#@AzureSDKTeam.onmicrosoft.com'
```

Comment:
Is it possible to hide the internal name?

Reply:
this domain name has to be correct otherwise user cannot be created.